applications/ParticleMechanicsApplication/python_scripts/conditions_python_utility.py

```diff
@@ -29,47 +29,46 @@ def Initialize(self, time_step):
 
     #
     def SetIncrementalDisp(self, time_step):
-
-        print('SONO NEL SET INCREMENTAL DISP')
+        print("Currently the SetIncremenetalDisp function is not active.")
```

Comment:
all comented?

Reply:
Previously it was also all commented, just changed the print to English. I will see how I can modify or remove the utility as soon as I checked it with @iiaconeta

Reply:
OK